docker-compose.yml

```diff
@@ -26,6 +31,10 @@ services:
     extends:
       file: docker-compose.apps.yml
       service: watch
+    extra_hosts:
+      # Both the browser and NextJS server (containerized) make requests to
+      # api.open.odl.local, forward those to host
+      - "api.open.odl.local:host-gateway"
```

Comment:
@gumaerc re
> When I try and load the page for my test program, I get "Error fetching course page metadata for programv1%3Aprogam-v1-test-program" (Yes, that is the readable ID of my test program). 

I suspect that was happening for all api-dependent metadata generation (though some of them may have fallbacks to static metadata w/o an api call), not just the mitxonline-specific ones.

This should fix it.